lib/rubocop/cop/style/one_line_conditional.rb

```diff
@@ -8,12 +8,17 @@ module Style
       class OneLineConditional < Cop
         include OnNormalIfUnless
 
-        MSG = 'Favor the ternary operator (?:) ' \
-              'over if/then/else/end constructs.'
+        MSG1 = 'Favor the ternary operator (?:) ' \
+               'over if/then/else/end constructs.'
+
+        MSG2 = 'Favor the ternary operator (?:) ' \
+               'over unless/then/else/end constructs.'
 
         def on_normal_if_unless(node)
-          return if node.loc.expression.source.include?("\n")
-          add_offense(node, :expression, MSG)
+          exp = node.loc.expression.source
+          return if exp.include?("\n") || !exp.include?('else')
```

Comment:
Checking if the source of the whole node contains the substring `else` is error-prone. Better to call `IfNode#if_else?`.